timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -294,47 +289,59 @@ def _check_data_for_evaluation(self, data: TimeSeriesDataFrame, name: str = "dat
                 f"all time series have length > prediction_length (at least {self.prediction_length + 1})"
             )
 
-    def _validate_num_val_windows(
+    @staticmethod
+    def _get_dataset_stats(data: TimeSeriesDataFrame) -> str:
+        ts_lengths = data.num_timesteps_per_item()
+        median_length = int(ts_lengths.median())
+        min_length = ts_lengths.min()
+        max_length = ts_lengths.max()
+        return (
+            f"{len(data)} rows, {data.num_items} time series. "
+            f"Median time series length is {median_length} (min={min_length}, max={max_length}). "
+        )
+
+    def _recommend_num_val_windows(self, train_data: TimeSeriesDataFrame, max_num_val_windows: int = 3) -> int:
```

Comment:
my personal opinion which is not backed by any data. Shall we set the `max_num_val_windows` default to 5 as the max allowed in Amazon Forecast is the 5 per the documentation: https://docs.aws.amazon.com/forecast/latest/dg/API_EvaluationParameters.html#forecast-Type-EvaluationParameters-NumberOfBacktestWindows

Reply:
I also don't have any data and I agree that 5 is a reasonable default, updated this parameter.